documentation/bin/createDocu.sh

```diff
@@ -18,7 +18,7 @@ do
 done
 
 export CLASSPATH_FILE='target/cp.txt'
-mvn clean dependency:build-classpath -Dmdep.outputFile=${CLASSPATH_FILE}
+mvn clean test dependency:build-classpath -Dmaven.test.failure.ignore=true  -Dmdep.outputFile=${CLASSPATH_FILE}
```

Comment:
Do you want to "skip the test execution" or to "ignore the test results"? The title is some how misleading

Reply:
First idea was to skip the execution, but looks like it is require. Therefore I switched to ignoring test failures. Updated the title.

Reply:
[clarification] The test execution is required since we generate a file which is used later on for retrieving the required plugins list for the docu.